index.css

```diff
@@ -0,0 +1,3 @@
+body {
+  color: white;
```

Comment:
This CSS rule is affecting the entire page, in such a way that all font colors are now white. This is preventing your user to see the labels of your input boxes. It is also affecting your list of books. Kindly change/remove this rule so your users can see the content of your page :blush: